extensions/wikia/AffiliateService/js/templates.js

```diff
@@ -13,14 +12,11 @@ define( 'ext.wikia.AffiliateService.templates', [], function() {
 
         if (options.showDisclaimer) {
             disclaimer = '<p class="aff-unit__disclaimer-message">'
-                + options.disclaimerMessage
-                + '</p>';
-        }
-
-        if (options.extraDisclaimer) {
-            extraDisclaimer = '<p class="aff-unit__extra-disclaimer-message">'
-                + options.extraDisclaimer
-                + '</p>';
+                + options.disclaimerMessage;
+                    if (options.extraDisclaimer) {
+						disclaimer += ' ' + options.extraDisclaimer
```

Comment:
This looks off (linting)

Reply:
My trial expired on WebStorm, which I had setup right for JS. Yep, that's right, I just blamed the tool I used, not the tool that I am... ;-)